disnake/user.py

```diff
@@ -277,7 +298,7 @@ class ClientUser(BaseUser):
 
         .. describe:: str(x)
 
-            Returns the user's name with discriminator.
+            Returns the user's username (with discriminator, if not migrated to new system yet).
```

Comment:
Thoughts on the following?
```suggestion
            Returns the user's username with discriminator if applicable.
```
Yours may be better.

Reply:
"if applicable" sounds fine but is a bit vague, I think I'd rather keep this explicit and unambiguous